### ISSUE TITLE
Standard mCIDE required for med_route ??? Medication Admin Continuous 

### ISSUE THREAD

**Issue by vaishvikr** [Common-Longitudinal-ICU-data-Format/CLIF#6]
![image](https://github.com/user-attachments/assets/e365f081-cd59-4d85-8aa0-ff6e2f985de8)
We have 131 med routes. Do we want only "e.g. IV, enteral" or More.
If more, is there mCIDE ???

**Comment by 08wparker** [comment]
@vaishvikr you are right we need `med_route_name` and `med_route_category`. @vaishvikr do you mind opening a pull request for https://github.com/clif-consortium/website and rename `med_route` to `med_route_name` and update the data dictionary to make this clear?

@kaveriC we can add `med_route_category` to the CDE to-do list

**Comment by 08wparker** [comment]
@rbaccile is working on the `med_route_category` CDE

**Comment by rbaccile** [comment]
![Capture](https://github.com/user-attachments/assets/572e87cd-67ad-4519-8d55-ee4605a900c6)

how should I categorize these routes? @08wparker

**Comment by clif-consortium** [comment]
Discuss post-ATS

**Comment by 08wparker** [comment]
the CDE list needs to be defined but the data dictionary is up to date

**Comment by deetherese** [comment]
Finalized route categories - 
1. Continuous - [mCIDE](https://github.com/Common-Longitudinal-ICU-data-Format/CLIF/blob/main/mCIDE/medication_admin_continuous/clif_medication_admin_continuous_med_route_categories.csv)
2. Intermittent - [mCIDE](https://github.com/Common-Longitudinal-ICU-data-Format/CLIF/blob/main/mCIDE/medication_admin_intermittent/clif_medication_admin_intermittent_med_route_categories.csv)
